CONTRIBUTING.md

```diff
@@ -27,6 +29,10 @@ blinking text"** a bug or a feature request?
 - Enhancements: The title should be an imperative statement of how things should be. **"Add support for blinking text"**
 - Bugs: The title should be a declarative statement of how things are. **"Text does not blink"**
 
+#### Logs
+
+Please provide sufficient logging around the issue which you are reporting as this will help with our investigation.
```

Comment:
"**DO NOT** include access tokens or other sensitive information. If you need to supply logs with sensitive information, supply them to developer support rather than posting them here; even when sending logs to developer support **DO NOT** include access tokens"